src/eventHandlers/checkSuite/index.spec.ts

```diff
@@ -17,13 +18,16 @@ const COMMIT_HEADLINE = 'Update test';
 const octokit = new GitHub('SECRET_GITHUB_TOKEN');
 const infoSpy = jest.spyOn(core, 'info').mockImplementation();
 const warningSpy = jest.spyOn(core, 'warning').mockImplementation();
+const getInputSpy = jest.spyOn(core, 'getInput').mockImplementation();
 
 describe('check Suite event handler', (): void => {
   it('does not log warnings when it gets triggered by Dependabot', async (): Promise<
     void
   > => {
     expect.assertions(1);
 
+    getInputSpy.mockReturnValueOnce('SQUASH');
+
```

Comment:
This could be moved to `beforEach` or be added as a `mockReturnValue` instead of `mockReturnValueOnce`  at the top of the file instead of in each test.

This comment is for all the test files where you add this in each test.

Reply:
nice catch!